src/components/exit-this-page/default/index.njk

```diff
@@ -0,0 +1,8 @@
+--- 
+title: Exit this page
+layout: layout-example.njk
+---
+
+{% from "govuk/components/exit-this-page/macro.njk" import govukExitThisPage %}
+
+{{govukExitThisPage({text: "Exit this Page"})}}
```

Comment:
This is a massive nitpick but could you add a single new line to the end of this file? This is standard procedure for all our files. There are technical benefits to this which I can explain if you want.